content/en/docs/tasks/configure-pod-container/static-pod.md

```diff
@@ -132,18 +132,18 @@ and store it on a web server so that you can pass the URL of that file to the ku
 
 Configure the kubelet on your selected node to use this web manifest by running it with `--manifest-url=<manifest-url>`
 
-    On Fedora, edit `/etc/kubernetes/kubelet` to include this line:
-
-    ```
-    KUBELET_ARGS="--cluster-dns=10.254.0.10 --cluster-domain=kube.local --manifest-url=<manifest-url>`
-    ```
-
-Now, restart the kubelet. On Fedora, you would run:
-
-    ```shell
-    # Run this command on the node where the kubelet is running
-    systemctl restart kubelet
-    ```
+> On Fedora, edit `/etc/kubernetes/kubelet` to include this line:
+>
+> ```
+> KUBELET_ARGS="--cluster-dns=10.254.0.10 --cluster-domain=kube.local --manifest-url=<manifest-url>`
+> ```
+>
+> Now, restart the kubelet. On Fedora, you would run:
+>
+> ```shell
+> # Run this command on the node where the kubelet is running
+> systemctl restart kubelet
+> ```
```

Comment:
Hi @diguage -- thanks for all the detail! Markdownguide.org's [blockquotes with other elements](https://www.markdownguide.org/basic-syntax/#blockquotes-with-other-elements) section says "Blockquotes can contain other Markdown formatted elements. **Not all elements can be used — you’ll need to experiment to see which ones work**."    So nesting code block syntax inside of blockquote syntax doesn't work with this version of Hugo.

Looking at the page as a whole, there is no reason to have spaces (or a tab?) at the start of lines 135-139. Probably the original author did a copy/paste from similar code in the bullet points (L90-102).  The spaces/tab at the beginning of each line 135-139 should be removed. See lines 150-175.

If you install [Hugo](https://gohugo.io/), you can [test your changes locally](https://kubernetes.io/docs/contribute/intermediate/#view-your-changes-locally). Hugo uses the [BlackFriday](https://github.com/russross/blackfriday) Markdown processor.

Reply:
I added a more detailed comment to your issue (#16510)

Reply:
Thanks a lot.

Remove the spaces may be a better way to resolve the problem.

I used to use the GitHub Format Markdown. Markdown has too many dialects. Let me learn the BlackFriday Markdown processor. 😆